spec/controllers/curation_concerns/generic_works_controller_spec.rb

```diff
@@ -238,55 +252,56 @@
   end
 
   describe '#destroy' do
-    let(:work_to_be_deleted) { create(:private_generic_work, user: user) }
```

Comment:
So I changed the name of this variable for consistency sake. Thats just that many more places you can miss a variable and confused me several times. So I figured it would be nice to go with a more consistent name. But I can change it back no biggy. Was just a proposition really.